src/components/PopupContent.vue

```diff
@@ -0,0 +1,75 @@
+<script lang="ts">
+import { defineComponent } from "vue";
+
+export default defineComponent({
+  name: "PopupContent",
+  props: {
+    onClick: {
+      type: Function,
+      required: true,
+    },
+    link: {
+      type: Boolean,
+      default: false,
+    },
+    value: {
+      type: String,
+      required: true,
+    },
+    color: {
+      type: String,
+      default: "normal",
+      validator: function (value: string) {
+        return ["normal", "danger"].includes(value);
+      },
+    },
+  },
+  emits: ["click"],
+});
+</script>
+
+<template>
+  <li
+    @click="$emit('click', value)"
+    :class="{
+      'popup-content': true,
+      [`popup-content--${color}`]: true,
+    }"
+  >
+    {{ value }}
+    <span v-show="link" class="material-icons popup-content__link">launch</span>
```

Comment:
```suggestion
    <span v-if="link" class="material-icons popup-content__link">launch</span>
```
`v-show`は`display: none`だった気がする
linkは描画後に値が変化しないのでv-ifでDOMを描画しない感じがよろしいかと（昔の知識）